doc/howto/use_cereal/load.hpp

```diff
@@ -0,0 +1,45 @@
+#include <fstream>
+#include <vector>
+
+//! [binary_include]
+#include <cereal/archives/binary.hpp> // includes the cereal::BinaryInputArchive and cereal::BinaryOutputArchive
+//! [binary_include]
+#include <cereal/types/vector.hpp>    // includes cerealisation support for std::vector
+
+#include <seqan3/core/concept/cereal.hpp>
```

Comment:
Why do we need this? I actually don't know.. Can you add a comment?

Reply:
We don't. :sweat_smile: